### PR TITLE
Add normalization module

### PR DESCRIPTION
**Pull request by roshankern**
@gwaybio This is ready for review!

Maybe it would be better to split the commands for running `raw_data_umaps.ipynb` and `normalize_data.py` into two `sh` files (or just put the commands in the readme). The `normalize_data.sh` name might be a bit confusing... what do you think?

### CODE REVIEW

File: 3.normalize_data/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+In this module, we use [UMAP](https://github.com/lmcinnes/umap) on control data to suggest batch effects are not the dominant signal in the mitosis movies.
+UMAP was introduced in [McInnes, L, Healy, J, 2018](https://arxiv.org/abs/1802.03426) as a manifold learning technique for dimension reduction.
+We use UMAP to reduce the feature data from 1280 features to 2 dimensions.
+
+As shown in [controls_Metadata_Gene.png](raw_data_umaps/controls_Metadata_Gene.png), UMAP tends to group features based on their gene.
+This demonstrates that the biological changes induced by gene pertubations have manifested in the `DeepProfiler` features extracted with `idrstream`.
+The other UMAPs in [raw_data_umaps/](raw_data_umaps/) suggest that batch effects from plate, well, and frame are not the dominant signal in the feature data.
+
+**Note:** UMAPs were generated with 10% of data from positive and negative controls.
```
Comment (gwaybio) [inline 3.normalize_data/README.md]:
```suggestion
**Note:** UMAPs were generated with 10% random subsample (without replacement) of data from positive and negative controls.
There are a total of ____ positive control cells and ____ negative control cells.
```

Accurate?

**Review by gwaybio — APPROVED** [review]
> Maybe it would be better to split the commands for running raw_data_umaps.ipynb and normalize_data.py into two sh files (or just put the commands in the readme). The normalize_data.sh name might be a bit confusing... what do you think?

I think how you have it is reasonable to me. You might remove the umap notebook and keep only the .py file. This is the necessary pipeline step (UMAP is supplementary 🍃 )

File: 3.normalize_data/README.md (hunk 1 above)
Comment (roshankern) [reply 3.normalize_data/README.md]: Accurate!

**Comment by roshankern** [comment]
> I think how you have it is reasonable to me. You might remove the umap notebook and keep only the .py file. This is the necessary pipeline step (UMAP is supplementary  )

I like the idea of only keeping the .py file. This is implemented in https://github.com/WayScience/mitocheck_data/pull/20/commits/dddfd17eafd8a8499577553963725f50c6fbbd92

**Pull request merged**